python/higgstotautauanalysiswrapper.py

```diff
@@ -160,23 +160,27 @@ def run(self):
 				print "compare baseline"
 				print config3.diff(config2)
 			### comparison end
-		
+
 		if exitCode < 256:
 			return exitCode
 		else:
 			return 1 # Artus sometimes returns exit codes >255 that are not supported
-		
+
 	def _initArgumentParser(self, userArgParsers=None):
-		
+
 		if userArgParsers is None:
 			userArgParsers = []
-		
+
 		self._parser = argparse.ArgumentParser(parents=[logger.loggingParser] + userArgParsers, fromfile_prefix_chars="@",
 		                                       description="Wrapper for Artus executables. Configs are to be set internally.")
-		
+
 		self._parser.add_argument("-x", "--executable", help="Artus executable. [Default: %(default)s]", default=os.path.basename(sys.argv[0]))
 		self._parser.add_argument("-a", "--analysis", required=True, help="Analysis nick [SM, MSSM] or import path ('HiggsAnalysis.KITHiggsToTauTau. ...' or 'HiggsAnalysis/KITHiggsToTauTau/python/ ... .py') of the config module.")
 
+		self._parser.add_argument("--sub-analysis", default='', action='store', choices=['btag-eff'], help="Keys to run a sub-analysis on top of base analyseis. Example: btag-egg Option to simplify the configs in order to estimate the efficiencies faster. [Default: %(default)s]")
+		self._parser.add_argument("--analysis-channels", default=['all'], nargs='+', type=str, choices=['all', 'mt', 'tt', 'et', 'ee', 'em', 'mm'], help="List of channels processed from the analysis. [Default: %(default)s]")
+		self._parser.add_argument("--no-svfit", default=False, action="store_true", help="Disable SVfit. Default: %(default)s]")
```

Comment:
Thanks for including the kwargs feature in config building.

Just a comment here on the option, that uses it:

I'd personally like more to have the the flags e.g. in the base config, rather than setting them via command line options. But for now it's fine as long as we don't overstretch it.